app/Console/Commands/InstallCommand.php

```diff
@@ -109,9 +111,9 @@ protected function configureDatabase(array $default = [])
         ], $default);
 
         $config['DB_DRIVER'] = $this->choice('Which database driver do you want to use?', [
-            'mysql'      => 'MySQL',
-            'postgresql' => 'PostgreSQL',
-            'sqlite'     => 'SQLite',
+            'mysql'  => 'MySQL',
+            'pgsql'  => 'PostgreSQL',
```

Comment:
@joecohens we could use `PDO::getAvailableDrivers` and ignore SQL Server to generate this?